webpack.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const path = require("path");
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+module.exports = {
+    entry: {
+        playground: path.resolve(__dirname, "src/playground/index.js")
+    },
+    output: {
+        filename: "[name].js",
+        path: path.resolve(__dirname, "_site/assets/js")
+    },
+    resolve: {
+        extensions: [".js", ".jsx"],
+        mainFields: ["browser", "main", "module"]
+    },
+    plugins: [
+        new NodePolyfillPlugin()
+    ],
+    devtool: "source-map",
+    module: {
+        rules: [
+            {
+                test: /\.jsx?$/u,
+                loader: "babel-loader",
+                include: path.resolve(__dirname, "src/playground"),
+                exclude: [
+                    path.resolve(__dirname, "node_modules"),
+                    path.resolve(__dirname, "src/playground/node_modules/lodash")
```

Comment:
why are we trying to exclude only the lodash inside the playground `node_modules` and not the whole `node_modules` itself?

Reply:
Actually, it was to match the [webpack config from eslint itself ](https://github.com/eslint/eslint/blob/3dd6741caa71c8cbe18df68eae63f2fd175f6d5d/webpack.config.js#L41). See [related discussion](https://github.com/eslint/archive-website/pull/665/files#r354108847). 

However, we r[ecently removed lodash](https://github.com/eslint/eslint/pull/14287), so I suppose we can update it here too.